core/server/models/member.js

```diff
@@ -30,6 +30,9 @@ const Member = ghostBookshelf.Model.extend({
         }, {
             key: 'products',
             replacement: 'products.slug'
+        }, {
+            key: 'newsletters',
+            replacement: 'newsletters.slug'
```

Comment:
are we happy to use slug for this (and products) given that it'll always be `default-newsletter`?

Reply:
🤔 Not sure i get it, this change is basically to expand a filter like `newsletters:abc` to internally map to `newsletters.slug:abc` since the actual value in filter will be the slug property of newsletter.

Reply:
So my question is: are we happy that the filter will always look like `newsletters:default-newsletter` given that there's no way to change the value of `default-newsletter`?